.github/workflows/upload-asset-on-release.yml

```diff
@@ -15,10 +22,24 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Validate version number
```

Comment:
We need to find a way to only run this version check if the release is NOT a pre-release. This will allow us to tag pre-releases without this workflow failing on us.

Reply:
Adding this to a task for later